x-pack/plugins/security_solution/cypress/e2e/detection_alerts/changing_alert_status.cy.ts

```diff
@@ -288,14 +262,14 @@ describe('Changing alert status', () => {
           selectNumberOfAlerts(numberOfAlertsToBeOpened);
 
           cy.get(SELECTED_ALERTS).should('have.text', `Selected ${numberOfAlertsToBeOpened} alert`);
-          cy.get(ALERTS_TREND_SIGNAL_RULE_NAME_PANEL).should('exist');
+          cy.get(ALERT_COUNT_TABLE_COLUMN(1)).should('exist');
```

Comment:
Can we be more specific about what ALERT_COUNT_TABLE_COLUMN(1) and  ALERT_COUNT_TABLE_COLUMN(3) really mean?